components/export/ui/src/main/resources/PhenoTips/ExportPreferences.xml

```diff
@@ -190,7 +190,8 @@
        'variants_evidence' : "$services.localization.render('phenotips.exportPreferences.field.variantsEvidence')",
        'variants_segregation' : "$services.localization.render('phenotips.exportPreferences.field.variantsSegregation')",
        'variants_sanger' : "$services.localization.render('phenotips.exportPreferences.field.variantsSanger')",
-       'variants_coordinates' : "$services.localization.render('phenotips.exportPreferences.field.variantsCoordinates')"
+       'variants_coordinates' : "$services.localization.render('phenotips.exportPreferences.field.variantsCoordinates')",
+       'diagnosis_notes' : "$services.localization.render('phenotips.exportPreferences.field.diagnosisNotes')"
```

Comment:
`diagnosis_notes` is also included in the autogenerated list, so now it appears twice, with different labels.